### PR TITLE
Add keplr integration for L2 devnet chain

### PR DESCRIPTION
**Pull request by natebeauregard**
Exposes the Cosmos [gRPC and API servers](https://docs.cosmos.network/v0.50/learn/advanced/grpc_rest) when starting a monomer node in process through the Monomer start command.

Also, creates a go script for bringing up a server for deploying Monomer devnet chain configs on Keplr.

## Summary by CodeRabbit

- **New Features**
	- Introduced a new integration for the Keplr wallet with support for Ethereum and Monomer Devnet.
	- Added a user interface for Keplr integration, featuring a styled button to enable the wallet.
  
- **Bug Fixes**
	- Enhanced error handling and graceful shutdown capabilities for the Monomer node.

- **Documentation**
	- Updated HTML and JavaScript files to improve user experience with the Keplr wallet integration.

- **Tests**
	- Adjusted server context configuration and error handling in tests to streamline processes.

### CODE REVIEW

File: integrations/integrations.go [hunk 1]
```diff
@@ -273,3 +294,104 @@ func startMonomerNode(
...
+	g *errgroup.Group,
+	config serverconfig.GRPCConfig,
+	clientCtx *client.Context,
+	svrCtx *server.Context,
+	app servertypes.Application,
+) (*grpc.Server, *client.Context, error) {
+	if !config.Enable {
+		// return grpcServer as nil if gRPC is disabled
+		return nil, clientCtx, nil
+	}
```
Comment (joshklop) [inline integrations/integrations.go:310]: let's move these lines outside the function.

File: integrations/integrations.go [hunk 2]
```diff
@@ -273,3 +294,104 @@ func startMonomerNode(
...
+	monomerCtx context.Context,
+	g *errgroup.Group,
+	svrCfg *serverconfig.Config,
+	clientCtx *client.Context,
+	svrCtx *server.Context,
+	app servertypes.Application,
+	home string,
+	grpcSrv *grpc.Server,
+) {
+	if svrCfg.API.Enable {
```
Comment (joshklop) [inline integrations/integrations.go:366]: let's move this check out side the function

File: integrations/integrations.go [hunk 3]
```diff
@@ -139,14 +150,24 @@ func startInProcess(
 		return fmt.Errorf("start Monomer node: %v", err)
 	}
 
+	// Add the tx service to the gRPC router if API or gRPC is enabled.
+	if svrCfg.API.Enable || svrCfg.GRPC.Enable {
+		app.RegisterTxService(*clientCtx)
+		app.RegisterTendermintService(*clientCtx)
+		app.RegisterNodeService(*clientCtx, *svrCfg)
```
Comment (joshklop) [inline integrations/integrations.go:157]: no need to check for errors?

Comment (natebeauregard) [reply integrations/integrations.go:157]: Nope, they all have no return vals

**Pull request merged**